packages/suite/src/components/suite/Banners/index.tsx

```diff
@@ -22,6 +23,11 @@ const Banners = () => {
     const online = useSelector(state => state.suite.online);
     const device = useSelector(state => state.suite.device);
     const { validMessages, dismissedMessages, config } = useSelector(state => state.messageSystem);
+    // The dismissal doesn't need to outlive the session. Use local state.
+    const [safetyChecksDismissed, setSafetyChecksDismissed] = useState(false);
+    useEffect(() => {
+        setSafetyChecksDismissed(false);
+    }, [device?.features?.safety_checks]);
```

Comment:
I have an idea about merged array/object of dismissed normal banners and message system banners here. 
To avoid duplicating useStates and useEffects when adding new normal banners.

Reply:
I was also thinking about tapping into the message system dismissal logic but then decided not to. Mainly because the messages' dismisalls are persistent, while the Safety checks dismissal should only persist for a session. The safety checks banner dismissing would have to become more complex - we'd have to come up with IDs for the beanners.[1] A small duplicity in the code and state seem like a smaller price to pay here. We can always abstract later. .)

[1] It may be that you have something simpler in your sleeve. I'm all ears. .)

Reply:
Okay, I agree with you as probably we will not add dozens of banners in the future which will duplicate this logic. Moreover, with more banners, it would be better to change the logic completely